### DISCUSSION TITLE
Fraction unbound (experiment) vs. Fraction unbound (plasma)

### DISCUSSION THREAD

**Discussion by Ember26** [Open-Systems-Pharmacology/Forum#1718] [Questions & Problems]
Hello!
I am referring to the reply to a discussion on the difference between the fraction unbound plasma vs experiment:
The `Fraction unbound (experiment)` is used for calculation of the specific renal clearance rate constant from renal clearance input value. So to correctly apply the measured or predicted renal clearance value to the model, the two `Fraction unbound` parameters should be equal. You should also consider this during parameter identification, if you are optimizing the fraction unbound.
_Originally posted by @PavelBal in https://github.com/Open-Systems-Pharmacology/Forum/discussions/1452#discussioncomment-4956564_

My question is: if the two fraction unbound values must be equal and I want to do parameter identification, I must select both of them as Identification Parameters. However, the optimized values may not be identical. Is there a way to force the two values to be identical?  
Looking forward to your reply. 
Thanks

**Comment by Yuri05** [comment]
If you want that some parameters are always identical during the parameter identification process, you must link them to the same **identification parameter**
For this, select all parameters of interest and then click the _upper_ "Add" button
(by clicking the _lower_ "Add" button you can link new simulation parameters to a previously defined identification parameter)
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/7e3500bc-0934-4ee3-b3aa-057905bc8d6e)

S. the documentation for details:
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#defining-identification-parameters
